### PR TITLE
Feature/version on front

### PR DESCRIPTION
**Pull request by RIN9A**
Added version on front page index

### CODE REVIEW

File: web/documentserver-example/python/templates/editor.html [hunk 1]
```diff
@@ -26,6 +26,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no" />
     <meta name="apple-mobile-web-app-capable" content="yes" />
     <meta name="mobile-web-app-capable" content="yes" />
+    <meta name="server-version" content="{{serverVersion}}">
+
```
Comment (aleksandrfedorov97) [inline web/documentserver-example/python/templates/editor.html:30]: need to add a version tag to index.html

Comment (aleksandrfedorov97) [inline web/documentserver-example/python/templates/editor.html:30]: empty line

File: web/documentserver-example/ruby/app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -34,6 +34,8 @@
   <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no, minimal-ui" />
   <meta name="apple-mobile-web-app-capable" content="yes" />
   <meta name="mobile-web-app-capable" content="yes" />
+  <meta name="server-version" content="<%= ConfigurationManager.new.version %>" />
+
```
Comment (aleksandrfedorov97) [inline web/documentserver-example/ruby/app/views/layouts/application.html.erb:38]: empty line

File: web/documentserver-example/php/templates/docEditor.tpl [hunk 3]
```diff
@@ -6,6 +6,9 @@
             maximum-scale=1, minimum-scale=1, user-scalable=no, minimal-ui" />
     <meta name="apple-mobile-web-app-capable" content="yes" />
     <meta name="mobile-web-app-capable" content="yes" />
+    <meta name="server-version" content="{serverVersion}">
```
Comment (aleksandrfedorov97) [inline web/documentserver-example/php/templates/docEditor.tpl:9]: need to add a version tag to index.tpl

File: web/documentserver-example/java-spring/src/main/resources/templates/editor.html [hunk 4]
```diff
@@ -5,6 +5,7 @@
         <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no, minimal-ui" />
         <meta name="apple-mobile-web-app-capable" content="yes" />
         <meta name="mobile-web-app-capable" content="yes" />
+        <meta name="server-version" th:content="${serverVersion}"/>
```
Comment (aleksandrfedorov97) [inline web/documentserver-example/java-spring/src/main/resources/templates/editor.html:8]: need to add a version tag to index.html

File: web/documentserver-example/java/src/main/webapp/editor.jsp [hunk 5]
```diff
@@ -1,6 +1,6 @@
 <%@page import="entities.FileModel"%>
+<%@page import="helpers.ConfigManager"%>
 <%@page contentType="text/html" pageEncoding="UTF-8"%>
-
```
Comment (aleksandrfedorov97) [inline web/documentserver-example/java/src/main/webapp/editor.jsp:3]: line deleted

File: web/documentserver-example/java/src/main/webapp/editor.jsp [hunk 6]
```diff
@@ -10,6 +10,7 @@
         <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no, minimal-ui" />
         <meta name="apple-mobile-web-app-capable" content="yes" />
         <meta name="mobile-web-app-capable" content="yes" />
+        <meta name="server-version" content="<%= ConfigManager.getProperty("version") %>" />
```
Comment (aleksandrfedorov97) [inline web/documentserver-example/java/src/main/webapp/editor.jsp:13]: need to add a version tag to index.jsp

File: web/documentserver-example/nodejs/views/editor.ejs [hunk 7]
```diff
@@ -6,6 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no" />
     <meta name="apple-mobile-web-app-capable" content="yes" />
     <meta name="mobile-web-app-capable" content="yes" />
+    <meta name="server-version" content="<%= serverVersion %>">
```
Comment (aleksandrfedorov97) [inline web/documentserver-example/nodejs/views/editor.ejs:9]: need to add a version tag to index.ejs

**Review by aleksandrfedorov97 — CHANGES_REQUESTED** [review]

**Review by sshakndr — CHANGES_REQUESTED** [review]
Put back 2nd line in editor.jsp file.

**Review by aleksandrfedorov97 — APPROVED** [review]

**Review by ZEROM22 — APPROVED** [review]

**Review by sshakndr — APPROVED** [review]

**Pull request merged**